frontend/src/react/components/documentation/Routes/Resources/DesktopAvailability.jsx

```diff
@@ -0,0 +1,157 @@
+import React from 'react';
+
+import Topic from './../../Topic.jsx';
+import Table from './../../Table.jsx';
+import Cell from './../../Cell.jsx';
+
+
+let codeExamples = {
+  python: `import requests
+
+params = {
+  "token": "uclapi-5d58c3c4e6bf9c-c2910ad3b6e054-7ef60f44f1c14f-a05147bfd17fdb",
+}
+r = requests.get("https://uclapi.com/resources/desktop_availability", params=params)
+print(r.json())`,
+
+  shell: `curl https://uclapi.com/resources/desktop_availability \
+-d token=uclapi-5d58c3c4e6bf9c-c2910ad3b6e054-7ef60f44f1c14f-a05147bfd17fdb
+`,
+
+  javascript: `fetch("https://uclapi.com/resources/desktop_availability?token=uclapi-5d58c3c4e6bf9c-c2910ad3b6e054-7ef60f44f1c14f-a05147bfd17fdb")
+.then((response) => {
+  return response.json()
+})
+.then((json) => {
+  console.log(json);
```

Comment:
Is this `console.log` needed?

Reply:
bump

Reply:
ye! how else do we print it to the screen? we do the same wiht python and shell